<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreProcessorGridCrosssectionWindow2</class>
 <widget class="QMainWindow" name="PreProcessorGridCrosssectionWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Crosssection Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../geodata/riversurvey/riversurvey.qrc">
    <normaloff>:/libs/geodata/riversurvey/images/iconRiverCrosssection.svg</normaloff>:/libs/geodata/riversurvey/images/iconRiverCrosssection.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="hSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PreProcessorGridCrosssectionWindow2::GraphicsView" name="graphicsView" native="true"/>
      <widget class="QSplitter" name="vSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="settingTable"/>
       <widget class="QTableView" name="editTable">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>View Setting Toolbar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="displayToolBar">
   <property name="windowTitle">
    <string>Display Setting ToolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreProcessorGridCrosssectionWindow2::GraphicsView</class>
   <extends>QWidget</extends>
   <header>subwindow/gridcrosssectionwindow2/public/preprocessorgridcrosssectionwindow2_graphicsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../geodata/riversurvey/riversurvey.qrc"/>
 </resources>
 <connections/>
</ui>
